File: C^C++/小鱼比可爱.cpp
//ÌâÄ¿Á´½Ó:"https://www.luogu.org/problemnew/show/P1428" 
#include<bits/stdc++.h>
using namespace std;
int main()
{
	int n;
	cin>>n;
	int a[105];
	for(int i=0;i<n;i++)
		cin>>a[i];
	int b[105]={0};
	for(int i=0;i<n;i++)
	{
		for(int j=0;j<i;j++)
			if(a[j]<a[i])
				b[i]++;
	}
	for(int i=0;i<n;i++)
		cout<<b[i]<<' ';
	return 0;
} 

File: JAVA/小鱼比可爱.java
//题目链接:"https://www.luogu.org/problemnew/show/P1428"
import java.util.*;
public class 小鱼比可爱 {
    public static void main(String[] args){
        Scanner s = new Scanner(System.in);
        int n = s.nextInt();
        int[] a = new int[105];
        int[] b = new int[105];
        for(int i=0;i<n;i++){
            a[i]=s.nextInt();
            for(int j=0;j<i;j++){
                if(a[j]<a[i])
                    b[i]++;
            }
        }
        for(int i=0;i<n;i++)
            System.out.printf("%d ", b[i]);
    }
}
